### ISSUE TITLE
DBT-Utils version number wrong in Readme.md

### ISSUE THREAD

**Issue by Austin1** [Teradata/dbt-teradata#60]
I think the version number may not be accurate in the DBT-Teradata ReadMe.MD file. It recommends 0.9.x as working with the current dbt-teradata, but after installing dbt-teradata, i'm receiving an error on run that dbt-utils version 0.9.6 is not compatible with the version of dbt-core that was installed by the dbt-teradata package.

```
Runtime Error
  Failed to read package: Runtime Error
    This version of dbt is not supported with the 'dbt_utils' package.
      Installed version of dbt: =1.1.0
      Required version of dbt for 'dbt_utils': ['>=1.2.0', '<2.0.0']
    Check for a different version of the 'dbt_utils' package, or run dbt again with --no-version-check
    
  
  Error encountered in /mnt/code/dbt_packages/dbt_utils/dbt_project.yml
```

If you go to the dbt-hub page for version 0.9.6, you'll see it requires 1.2.0:

https://hub.getdbt.com/dbt-labs/dbt_utils/0.9.6/

It seems the first version supporting the dbt-teradata?

**Comment by VarunSharma15** [comment]
Hi @Austin1,
We released the new version of dbt-teradata 1.2.0 yesterday, which is compatible with dbt-utils 0.9.6.
Please try to upgrade the adapter to 1.2.0 and please let us know if you face any more issues.
https://pypi.org/project/dbt-teradata/

**Comment by Austin1** [comment]
Thanks, this is working. Appreciate it!
